Ggplot theme update

SRSchmittling opened the pull request:
added recipe to compare modifying a ggplot theme with theme_update vs. theme_replace

SRSchmittling committed:
Draft 1

using ggplot theme_update vs. theme_replace

SRSchmittling committed:
Rename UpdateRTheme.md to R/UpdateRTheme.md

SRSchmittling committed:
Draft 2 [Test Uploading Again]

SRSchmittling committed:
finish recipe

SRSchmittling committed:
Update README.md

Add recipe to README.md

HunterJiang97 reviewed (changes requested):
Hi Selene,

Thanks for uploading this, here are some ideas from my end:

1. In your SetRTheme.md, it may be better to use a simpler ggplot command (one-or-two lines, the shorter the better) so we can concentrate on the main part, setting the theme. (Also, your code cannot run on my local PC)

2. Also in the SetRTheme.md, I would love some consistency, that is: it would be much better to include some result figures just like UpdateRTheme.md.

Best,
Hunter

SRSchmittling commented:
changed set_theme() to theme_set() so code runs. I added images.  I didn't change the type of plots because I want enough elements in the plots to sufficiently so differences in themes.

SRSchmittling committed:
Updates to SetRTheme as requested by HJ on github.
